README.md

```diff
@@ -20,7 +20,7 @@ Application-level tracing for Rust.
 [gitter-url]: https://gitter.im/tokio-rs/tracing
```

Comment:
Can we also add `https://img.shields.io/badge/docs-master-blue` with a link to the master docs, preferably after the docs.rs badge?

Reply:
Done, added to all crates with a README.md.

Reply:
Done, added to all crates with a README.md.